src/kernel/src/arch/aarch64/memory/pagetables/consistency.rs

```diff
@@ -0,0 +1,152 @@
+// use alloc::boxed::Box;
+
+use crate::{
+    arch::address::{PhysAddr, VirtAddr},
+    // interrupt::Destination,
+};
+
+// TODO:
```

Comment:
Is there still a TODO to TODO here?

Reply:
The scattered TODOs are reminders to actually implement certain things for ARM. For now the code has a lot of stubs and placeholder values